reverse_proxy_handler.cc

```diff
@@ -0,0 +1,51 @@
+#include "reverse_proxy_handler.h"
+
+ReverseProxyHandler::ReverseProxyHandler() {}
+
+ReverseProxyHandler::~ReverseProxyHandler() {}
+
+ReverseProxyHandler::Status ReverseProxyHandler::Init(const std::string& uri_prefix, const NginxConfig& config) {
+    uri_prefix_ = uri_prefix;
+
+    bool remote_host_found = false;
+    bool remote_port_found = false;
+
+    // expecting two statements in child block: remote_host <remote_host>;
+    //                                          remote_port <remote_port>;
```

Comment:
The same comment here, let's come back to this once they actually update the spec with further details.